### DISCUSSION TITLE
Centered tables from code using typst

### DISCUSSION THREAD

**Discussion by psads-git** [quarto-dev/quarto-cli#8768] [Q&A]
Consider the following `quarto` document. If `format` is `pdf`, the output is a centered table. However, when `format` is `typst`, the resulting table is left-aligned. Is it possible to get the table centered with `format: typst` likewise as with `format: pdf`? Thanks!

````
---
format: typst
papersize: a4
fontsize: 12pt
margin:
  x: 2.54cm
  y: 2.54cm
jupyter: python3
---

```{python}
import seaborn as sns

iris = sns.load_dataset('iris')
iris.head()
```
````

**Comment by mcanouil** [comment]
Closing in favour of:
- #8797
